src/v3/test/integration/__snapshots__/authenticator-piv-cac-verification.test.tsx.snap

```diff
@@ -62,7 +62,6 @@ exports[`authenticator-piv-cac-verification renders form with custom PIV/CAC But
                     class="MuiButton-startIcon MuiButton-iconSizeMedium emotion-12"
                   >
                     <img
-                      alt="PIV Card"
```

Comment:
These should get fixed once you fix the comment above with the iconAlt property.